hail/python/hailtop/batch/backend.py

```diff
@@ -329,6 +329,9 @@ def transfer_dicts_for_resource_file(res_file: Union[resource.ResourceFile, reso
                 code += ['\n']
 
                 run_code(code)
+
+                for job in batch._unsubmitted_jobs:
+                    job._submitted = True
```

Comment:
I think this is indented one too deep. Regardless, this is already in a loop over `batch._unsubmitted_jobs`, so can we just nix this loop and put `job._submitted = True` after `run_code(code)`?